{{cookiecutter.project_slug}}/clients/vue3/src/views/Home.vue

```diff
@@ -1,7 +1,7 @@
 <template>
-  <div class="home">
+  <div class="home flex flex-col items-center">
     <img alt="Project Logo" src="../assets/logo.png" />
-    <h2>Welcome to {{cookiecutter.project_name}}!</h2>
+    <h1>Welcome to {{cookiecutter.project_name}}!</h1>
```

Comment:
**Info:** If we're going to be using the heading tags, we should always start with `<h1>` for a11y reasons. Formatting can be adjusted as needed with CSS.